### ISSUE TITLE
Cannot open context menu with keyboard for blobs and messages

### ISSUE THREAD

**Issue by v-xuanzh** [microsoft/AzureStorageExplorer#1294]
**Storage Explorer Version:** 1.7.0_20190416.1
**Platform/OS:** Linux Ubuntu 18.04
**Architecture:** x64
**Commit:** c684556c
**Regression From:** Not a regression

**Steps to reproduce:**
1. Expand any one account -> Blob Containers.
2. Open one blob container which contains one blob at least -> Focus on one blob then press the shortcut key(Between the Windows and Control)
3. Check whether the context menu shows up or not.

**Expect Experience:**

The context menu shows up.

**Actual Experience:**

Nothing happens.

**More info:**

1. This issue also reproduces for messages of queue.
2. This issue doesn't reproduce for files(File Share) and entities(Tables).

**Comment by JasonYeMSFT** [comment]
Track this at #1825.
